Makefile

```diff
@@ -22,4 +22,4 @@ deps: submodules
 	fi
 
 test: deps
-	export LLVM_SYS_170_PREFIX=$(LLVM_PATH) && cd era-compiler-tester && cargo run --verbose --features lambda_vm --release --bin compiler-tester -- --path  tests/solidity/simple/ --target EraVM
+	export LLVM_SYS_170_PREFIX=$(LLVM_PATH) && cd era-compiler-tester && cargo run --verbose --features lambda_vm --release --bin compiler-tester -- --path  tests/solidity/simple/ --target EraVM --mode "Y+M3B3 0.8.26"
```

Comment:
Why is this flag being added?

Reply:
Because without the flag it takes 6 hours to run the CI, which is a lot for PRs.

Reply:
We should only run all modes on the [periodical CI job](https://github.com/lambdaclass/era_vm/pull/97) to prevent long CIs for pull requests, it also saves time while testing locally.